test/jasmine/tests/mapbox_test.js

```diff
@@ -449,7 +449,7 @@ describe('mapbox plots', function() {
 
             var divStyle = mapInfo.div.style;
             ['left', 'top', 'width', 'height'].forEach(function(p, i) {
-                expect(parseFloat(divStyle[p])).toBeWithin(dims[i], 5);
+                expect(parseFloat(divStyle[p])).toBeWithin(dims[i], 8);
```

Comment:
other than the occasional
```
Uncaught TypeError: gd.emit is not a function
  at /var/folders/z4/src/plots/mapbox/mapbox.js:147:0
```
that still causes `test-jasmine` to fail intermittently